src/components/molecules/card/card.scss

```
@@ -0,0 +1,45 @@
+@use "../../variables" as *;
+.card {
+  border-radius: 0%;
+  border: none;
+  &::before {
+    position: absolute;
+    border-bottom: solid 150px transparent;
+    border-left: solid 150px white;
+    content: "";
+    z-index: 2;
+  }
+  .card-title {
+    padding-left: 6px;
+    position: absolute;
+    color: $orange_bistre;
+    font-size: $font-sizeL;
+    font-weight: bold;
+    margin: 0;
+    z-index: 2;
+  }
+
+  .card-wrapper {
+    overflow: hidden;
+
+    .card-img-top {
+      transition: ease 0.5s;
+
+      &:hover {
+        transform: scale(1.2);
+      }
+    }
+  }
+}
+
+.card-body {
+  padding: 32px;
+  background-color: $white-smoke;
+
+  .card-text {
+    margin-top: 32px;
+    line-height: 33px;
+    font-size: $font-sizeS;
+    text-align: center;
+  }
+}

```

HTMLを見ると、.card-body は .card の子要素になっていると思うので、その構造をscssの入れ子として表現してください。
```suggestion
.card {
~~~ 中略
  .card-body {
    padding: 32px;
    background-color: $white-smoke;
  
    .card-text {
      margin-top: 32px;
      line-height: 33px;
      font-size: $font-sizeS;
      text-align: center;
    }
  }
}

```